valhalla/app/javascript/components/Thumbnails.vue

```diff
@@ -105,13 +137,34 @@ export default {
         return false
       }
     },
+    isCut: function (thumbnail) {
+      if (this.cut.indexOf(thumbnail) > -1) {
+        return true
+      } else {
+        return false
+      }
+    },
     isSelected: function (thumbnail) {
       if (this.selected.indexOf(thumbnail) > -1) {
         return true
       } else {
         return false
       }
     },
+    isCutDisabled: function () {
```

Comment:
Might this be refactored into an arrow function expression?  (It seems to me that this is something similar to just returning `!!this.cut.length`)

Reply:
I will look at it. At the very least, I could reduce line numbers by using ternary operators in all of those types of functions.

Reply:
@jrgriffiniii I was having trouble using arrow functions in this context... the Vue docs use the current style. However, I did reduce the if/then cruft, making the functions much shorter. I also fixed a couple of cut/paste bugs that I didn't notice. This should be ready to merge.